consensus/ibftlegacy/src/main/java/org/hyperledger/besu/consensus/ibftlegacy/IbftBlockHashing.java

```diff
@@ -114,15 +115,15 @@ public static Address recoverProposerAddress(
         .collect(Collectors.toList());
   }
 
-  private static BytesValue encodeExtraDataWithoutCommittedSeals(
+  private static Bytes encodeExtraDataWithoutCommittedSeals(
       final IbftExtraData ibftExtraData, final Signature proposerSeal) {
     final BytesValueRLPOutput extraDataEncoding = new BytesValueRLPOutput();
     extraDataEncoding.startList();
     extraDataEncoding.writeList(
-        ibftExtraData.getValidators(), (validator, rlp) -> rlp.writeBytesValue(validator));
+        ibftExtraData.getValidators(), (validator, rlp) -> rlp.writeBytes(validator.toBytes()));
```

Comment:
Again, I thought using the delegated bytes we could avoid the `toBytes()` call

Reply:
you're right, and I got rid of all those.